enterprise/server/remote_execution/containers/ociruntime/ociruntime.go

```diff
@@ -440,9 +446,15 @@ func (c *ociContainer) Run(ctx context.Context, cmd *repb.Command, workDir strin
 		return commandutil.ErrorResult(status.UnavailableErrorf("create OCI bundle: %s", err))
 	}
 
-	return c.doWithStatsTracking(ctx, func(ctx context.Context) *interfaces.CommandResult {
+	res := c.doWithStatsTracking(ctx, func(ctx context.Context) *interfaces.CommandResult {
 		return c.invokeRuntime(ctx, nil /*=cmd*/, &interfaces.Stdio{}, 0 /*=waitDelay*/, "run", "--bundle="+c.bundlePath(), c.cid)
 	})
+	if res.ExitCode == ociSIGKILLExitCode || res.ExitCode == ociExecSIGTERMExitCode {
+		res.ExitCode = commandutil.KilledExitCode
+		res.Error = commandutil.ErrSIGKILL
+	}
```

Comment:
I think there is a chance this will break people if they are already returning these exit codes for another reason.

Is there anything logged by `crun` that we could maybe use to distinguish these scenarios?

Reply:
no, it's pretty barebone :(

https://github.com/containers/crun/blob/283ffdf7dc0e92f61d78df362f08e42ba11efa7a/src/libcrun/utils.h#L415-L424